### PR TITLE
Update DOCs Dutch TESTed

### PR DESCRIPTION
**Pull request by bsels**
- [x] Review existing docs
- [x] Update structure
  - [x] Update Homepage
  - [x] Add config single programming language
  - [x] TESTed without Dodona

### CODE REVIEW

File: nl/tested-judge/template-exercise/README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
-title: "Oefeningssjabloon TESTed"
-description: "Tutorial: Oefeningssjabloon TESTed"
+title: "Meertalige programmeeroefeningen"
+description: "Tutorial: mMertalige programmeeroefeningen"
```
Comment (pdawyndt) [inline nl/tested-judge/template-exercise/README.md:3]: Meertalige

File: nl/tested-judge/dsl/README.md [hunk 2]
```diff
@@ -948,15 +968,24 @@ voorlopig niet ondersteund.
...
-```javascript
+```tested
 new Counter()
 new object.Pair("Paar", 8.4e-5)
 ```
 
+### Globale variabelen
+
+Een **globale variabele** is een eigenschap van de globale namespace.
+Een **globale variabele** wordt genoteerd als een naam genoteerd tussen scheve haken.
```
Comment (niknetniko) [inline nl/tested-judge/dsl/README.md:979]: According to Wikipedia, these are called "punthaakjes, hoekankers, hoekhaakjes", but unsure what the proper term for these is in Dutch.

File: nl/tested-judge/standalone/README.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
+---
+title: TESTed zonder Dodona
+description: "TESTed zonder Dodona"
+---
+
+# TESTed zonder Dodona
+Hoewel tijdens de ontwikkeling van TESTed er gefocust werd voor het gebruik op het Dodona-platform, kan TESTed ook los
+van Dodona gebruikt worden.
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:8]:
```suggestion
Hoewel er tijdens de ontwikkeling van TESTed gefocust werd voor het gebruik met het Dodona-platform, kan TESTed ook los
van Dodona gebruikt worden.
```

File: nl/tested-judge/standalone/README.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+In deze handleiding zullen we uitleg hoe je TESTed kan gebruiken los van het Dodona-platform.
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:9]:
```suggestion
In deze handleiding zullen we uitleggen hoe je TESTed kan gebruiken los van het Dodona-platform.
```

File: nl/tested-judge/standalone/README.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+## 1. Klonen van de TESTed GitHub repository
+De eerst stap om TESTed los van Dodona te gebruiken is het klonen of downloaden van de
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:12]:
```suggestion
De eerst stap is het klonen of downloaden van de
```

File: nl/tested-judge/standalone/README.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+[TESTed GitHub repository](https://github.com/dodona-edu/universal-judge).
+
+![Klonen TESTed GitHub repository](./klonen_tested_repo.png)
+
+## 2. Installeren software
+De volgende stap is het installeren van de afhankelijkheden van de TESTed judge.
+
+::: tip Windows 10 tip
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:20]:
```suggestion
::: tip Tip voor Windows 10
```

File: nl/tested-judge/standalone/README.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+Windows 10 gebruikers raden we aan om het [Windows Subsystem for Linux](https://ubuntu.com/wsl) te gebruiken.
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:21]:
```suggestion
Gebruikers van Windows 10 raden we aan om het [Windows Subsystem for Linux](https://ubuntu.com/wsl) te gebruiken.
```

File: nl/tested-judge/standalone/README.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+:::
+
+### 1) Installeren kern software
+De kern van de TESTed judge maakt gebruikt van [Python 3.9](https://docs.python.org/3/whatsnew/3.9.html).
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:25]:
```suggestion
De kern van de TESTed maakt gebruikt van [Python 3.9](https://docs.python.org/3/whatsnew/3.9.html).
```

File: nl/tested-judge/standalone/README.md [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+Hiervoor moet het `python3.9` commando beschikbaar zijn vanuit het `PATH`.
+
+TESTed heeft een aantal Python dependencies die met behulp van het
+[pip](https://en.wikipedia.org/wiki/Pip_(package_manager)) commando vanuit de root-directory van TESTed:
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:29]: I would link to the pip docs

File: nl/tested-judge/standalone/README.md [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+![Klonen TESTed GitHub repository](./klonen_tested_repo.png)
+
+## 2. Installeren software
+De volgende stap is het installeren van de afhankelijkheden van de TESTed judge.
+
+::: tip Windows 10 tip
+Windows 10 gebruikers raden we aan om het [Windows Subsystem for Linux](https://ubuntu.com/wsl) te gebruiken.
+:::
+
+### 1) Installeren kern software
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:24]: I would mention somewhere that this section is a relatively standard Python project (python & pip & requirements.txt)

File: nl/tested-judge/standalone/README.md [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+> pip3 install -r requirements-test.txt
+```
+
+### 2) Installeren specifieke programmeertaal
+:::: tabs
+::: tab Bash
+De Bash-oefening vereisen dat het [bash](https://nl.wikipedia.org/wiki/Bash) commando geïnstalleerd is en
+beschikbaar is vanuit het `PATH`.
+
+Bash is standaard geïnstalleerd op de Debian besturingssystemen en Mac OS.
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:45]: I wouldn't mention this; there are a lot of linux distribution, and I doubt someone will change distributions for tested.

File: nl/tested-judge/standalone/README.md [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+npm install -g abstract-syntax-tree@2.17.6
+```
+
+De JavaScript-linter [eslint](https://eslint.org/) moet ook beschikbaar zijn in het `PATH`,
+wanneer je linting wenst te gebruiken.
+:::
+::: tab Kotlin
+De Kotlin-oefeningen vereisen dat de `kotlin` en `kotlinc` commando's beschikbaar zijn in het `PATH`.
+De installatiemogelijkheden kunnen teruggevonden worden op de
+[Kotlin website](https://kotlinlang.org/docs/command-line.html).
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:89]:
```suggestion
[Kotlin-website](https://kotlinlang.org/docs/command-line.html).
```

File: nl/tested-judge/standalone/README.md [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+:::
+::: tab Java
+De Java-oefeningen vereisen dat Java 11 geïnstalleerd is.
+Zowel de [Oracle versie](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+als de [OpenJDK versie](https://openjdk.java.net/projects/jdk/11/) kan gebruikt worden.
+
+De commando's `javac` en `java` moeten beschikbaar zijn in het `PATH`.
+
+Java gebruikt de *Checkstyle-8.41* linter, dit is een JAR-bestand die gedownload kan worden op de
+[Checkstyle GitHub pagina](https://github.com/checkstyle/checkstyle/releases/download/checkstyle-8.41/checkstyle-8.41-all.jar).
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:70]:
```suggestion
[GitHub-pagina van Checkstyle](https://github.com/checkstyle/checkstyle/releases/download/checkstyle-8.41/checkstyle-8.41-all.jar).
```

File: nl/tested-judge/standalone/README.md [hunk 14]
```diff
@@ -0,0 +1,170 @@
...
+```bash
+> cabal v1-install aeson
+```
+
+De Haskell-linter [hlint](https://hackage.haskell.org/package/hlint) moet ook beschikbaar zijn in het `PATH`,
+wanneer je linting wenst te gebruiken.
+:::
+::: tab Java
+De Java-oefeningen vereisen dat Java 11 geïnstalleerd is.
+Zowel de [Oracle versie](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:64]:
```suggestion
Zowel de [Oracle-editie](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
```

File: nl/tested-judge/standalone/README.md [hunk 15]
```diff
@@ -0,0 +1,170 @@
...
+als de [OpenJDK versie](https://openjdk.java.net/projects/jdk/11/) kan gebruikt worden.
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:65]:
```suggestion
als de [OpenJDK-editie](https://openjdk.java.net/projects/jdk/11/) kan gebruikt worden.
```

File: nl/tested-judge/standalone/README.md [hunk 16]
```diff
@@ -0,0 +1,170 @@
...
+De JavaScript-linter [eslint](https://eslint.org/) moet ook beschikbaar zijn in het `PATH`,
+wanneer je linting wenst te gebruiken.
+:::
+::: tab Kotlin
+De Kotlin-oefeningen vereisen dat de `kotlin` en `kotlinc` commando's beschikbaar zijn in het `PATH`.
+De installatiemogelijkheden kunnen teruggevonden worden op de
+[Kotlin website](https://kotlinlang.org/docs/command-line.html).
+
+Kotlin gebruikt de *KTLint* linter, dit is een JAR-bestand die gedownload kan worden op de
+[KTLint GitHub pagina](https://github.com/pinterest/ktlint/releases/download/0.41.0/ktlint).
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:92]:
```suggestion
[GitHub-pagina voor KTLint](https://github.com/pinterest/ktlint/releases/download/0.41.0/ktlint).
```

File: nl/tested-judge/standalone/README.md [hunk 17]
```diff
@@ -0,0 +1,170 @@
...
+:::
+::::
+
+## 3. Evalueren oefeningen
+TESTed kan uitgevoerd met behulp van het volgende commando in de TESTed root-directory:
+```bash
+> python3 -m tested
+```
+
+Dit commando vereist een JSON-document als standaardinvoer met de volgende velden:
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:113]: Perhaps we can link to the config file here? I thought this was the same structure, but unsure

File: nl/tested-judge/standalone/README.md [hunk 18]
```diff
@@ -0,0 +1,170 @@
...
+| Optie | Beschrijving |
+| ------| ------------ |
+| `time_limit` | De maximale tijdslimiet die gebruikt mag worden tijdens de evaluatie. |
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:116]:
```suggestion
| `time_limit` | De tijdslimiet die gebruikt mag worden tijdens de evaluatie. |
```

File: nl/tested-judge/standalone/README.md [hunk 19]
```diff
@@ -0,0 +1,170 @@
...
+| `memory_limit` | De maximale geheugenlimiet die gebruikt mag worden tijdens de evaluatie. |
```
Comment (niknetniko) [inline nl/tested-judge/standalone/README.md:117]:
```suggestion
| `memory_limit` | De geheugenlimiet die gebruikt mag worden tijdens de evaluatie. |
```

**Review by niknetniko — CHANGES_REQUESTED** [review]

File: nl/tested-judge/standalone/README.md (hunk 17 above)
Comment (bsels) [reply nl/tested-judge/standalone/README.md:113]: This document contains only the `evaluation` field of the config file.

File: nl/tested-judge/dsl/README.md (hunk 2 above)
Comment (bsels) [reply nl/tested-judge/dsl/README.md:979]: "Scheve haken" is a proper term.
See page 29 on https://inigem.ugent.be/jvlfx/jvlfx.pdf.

Comment (niknetniko) [reply nl/tested-judge/dsl/README.md:979]: I don't know where prof. Coolsaet found this term, but I did some research, and I can't find a lot of usage of "scheve haken" for "<" ">". Vandale uses "punthaakjes" or "vishaakjes", so I would suggest taking one of those terms.

Comment (bsels) [reply nl/tested-judge/dsl/README.md]: I changed it in "punthaakjes".

**Review by niknetniko — APPROVED** [review]

**Review by pdawyndt — APPROVED** [review]

**Pull request merged**